src/i18n/locales/en.json

```diff
@@ -76,17 +81,17 @@
   "stop-button-title": "Stop recording",
   "upload-notification": "Currently uploading...",
   "upload-settings-button-store": "Save connection settings",
-  "upload-settings-modal-header": "Opencast Connection",
+  "upload-settings-modal-header": "Podcast Connection",
   "upload-settings-label-server-url": "Server URL",
   "upload-settings-label-username": "Username",
   "upload-settings-label-password": "Password",
   "upload-settings-invalid-url": "This is not a valid URL",
   "upload-settings-invalid-url-http-start": "The URL should start with 'https://' or 'http://'",
-  "upload-settings-error-server-unreachable": "The server you specified is not reachable. Possible causes: the server URL you specified is incorrect; your device is offline; the Opencast server is offline; the Opencast server is not correctly configured to allow CORS requests from Opencast Studio.",
-  "upload-settings-error-response-not-ok": "The server returned a non-OK answer to our request. This likely means that the server you specified is not a valid Opencast server.",
-  "upload-settings-error-invalid-response": "The server returned an invalid response to our request. This likely means that the server you specified is not a valid Opencast server.",
-  "upload-settings-invalid-login-data": "Could not log into Opencast server: the username/password you provided are invalid.",
-  "upload-settings-invalid-provided-login": "The server specifies that it provides an automatic login, but the request returned non-authenticated. This is either an internal bug of Opencast Studio or your system administrator misconfigured the server. There is nothing you can do :(",
+  "upload-settings-error-server-unreachable": "The server you specified is not reachable. Possible causes: the server URL you specified is incorrect; your device is offline; the Podcast server is offline; the Podcast server is not correctly configured to allow CORS requests from Podcast Studio.",
```

Comment:
Discard these error messages changes if the user does not see them because they are the settings page.